api/policies/CheckPermissions.js

```diff
@@ -33,14 +33,29 @@ module.exports = class CheckPermissionsPolicy extends Policy {
           if (action !== 'create' && permission[0].relation === 'owner') {
             if (action === 'access' || !req.params.id) {
               if (action === 'update' || action === 'destroy') {
-                return next({
-                  message: 'Update and Destroy are not yet supported with permissions, please do your request manually',
-                  statusCode: 400,
-                  code: 'E_UNSUPPORTED'
+                this.app.services.FootprintService.find(modelName, {}, {
+                  populate: [{
+                    model: this.app.orm.User,
+                    as: 'owners',
+                    required: true,
+                    where: {id: req.user.id}
+                  }]
+                }).then(items => {
+                  if (items.length > 1000) this.app.log.warning('Caution, bulk update/destroy for large model sets can be unperformant')
+                  const ids         = items.reduce((acc, val) => {
+                    acc.push(val.id)
+                    return acc
+                  }, [])
+                  req.query.where = {
```

Comment:
Does this work ? ^^ maybe only with sequelize but as it's the only one supported for now it's fine :) 
Anyway like this you ignore every other params sent by the user as criteria, you need to retrieve them and merge them here you can use https://github.com/trailsjs/trailpack/blob/master/server.js#L47 on express trailpack